src/firebase-app.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import { Credential, GoogleOAuthAccessToken, getApplicationDefault } from './auth/credential';
+import { getApplicationDefault } from './auth/credential';
```

Comment:
Can we not touch the `firebase-app` module just yet? Is it possible to refactor only the database module in this PR and update the app module in a separate PR?